sym-lib-table

```diff
@@ -1,3 +1,3 @@
 (sym_lib_table
-  (lib (name test)(type Legacy)(uri ${KIPRJMOD}/test.lib)(options "")(descr ""))
+  (lib (name "test")(type "KiCad")(uri "${KIPRJMOD}/test.kicad_sym")(options "")(descr ""))
```

Comment:
Did KiCad generate the quotes for the name, type, and uri fields or were they added manually?

`fp-lib-table` doesn't use quotes for these fields. Should it be updated to use quotes for consistency with `sym-lib-table`?

Reply:
The quotes were automatically generated when I migrated the symbols library over within KiCad 6.

As a test, I removed the file and had KiCad regenerate it, by re-adding the library from `Manage Symbol Libraries` within KiCad. The newly generated file was identical to the previous one, with the quotes in the same places.